docs/installation/linux/debian.md

```diff
@@ -21,6 +21,12 @@ Docker is supported on the following versions of Debian:
  >**Note**: If you previously installed Docker using `APT`, make sure you update
  your `APT` sources to the new `APT` repository.
 
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using Debian-managed packages, consult your
+Debian documentation. If you are comfortable with package managers, and are familiar
+with Docker's prerequisites, feel free to skip to step 3 in the [Install](https://docs.docker.com/engine/installation/debian/#install-docker) section
```

Comment:
The # url id selector doesn't work for me on any of the links except the centos one, looks like the element id has some sort of hash value appended to it.